plugins/carbon_black_response/plugin.spec.yaml

```diff
@@ -766,6 +804,7 @@ actions:
         description: The list of processes
         type: '[]process'
```

Comment:
Same again, here and the above/below fields. Won't let me comment on them directly but go through all the inputs/outputs in the actions section of the spec file and give them a title like: `title: Processes` to match the `processes:` part 

Only asking because it seems the titles used to exist in all the schema files and now they've disappeared since you refreshed and the titles don't exist

Probably an issue from switching from `icon-plugin` to `insight-plugin`